packages/create-package/template/vite.config.js

```diff
@@ -0,0 +1 @@
+export { default } from "@ui5/webcomponents-tools/components-package/vite.config.js"; //eslint-disable-line
```

Comment:
can you make it a separate variable and put a comment how it can be extended, then the export will also be sepeate